ecosystem/node/run-validator.mdx

```diff
@@ -0,0 +1,670 @@
+
+- All outgoing connections are to be open.
+- A static external IP address.
+- One UDP port open for incoming connections.
+
+To determine which UDP ports are open, use the following command:
+
+```bash
+netstat -tulpn
+```
+
+### 1.5 Follow network announcements
+
+Subscribe and follow the instructions provided for validators with Telegram channels.
+
+<Aside type="tip">
+  Do not ignore announcements for TON Testnet; validators that violate the maintenance rules are fined and excluded from the network.
+</Aside>
+
+| Channel                                      | Network     |
+| -------------------------------------------- | ----------- |
+| [@tonstatus](https://t.me/tonstatus)         | TON Mainnet |
+| [@testnetstatus](https://t.me/testnetstatus) | TON Testnet |
+
+### 1.6 Prepare validator user
+
+Prepare and log in a user for the validator.
+
+1. Create a non-root user
+
+```bash
+# Create a non-root validator user
+sudo adduser <USERNAME>
+sudo usermod -aG sudo <USERNAME>
+```
+
+2. Switch to it by reconnecting to the server via `ssh`
+
+```bash
+#reconnect as the new user
+exit
+ssh <USERNAME>@<SERVER_IP>
+```
+
+## Step 2: Validator node installation
+
+### 2.1 Download MyTonCtrl
+
+Run:
+
+```shell
+wget https://raw.githubusercontent.com/ton-blockchain/mytonctrl/master/scripts/install.sh
+```
+
+<Image
+  src="/resources/images/nodes/run-validator/run-validator-4.png"
+  darkSrc="/resources/images/nodes/run-validator/run-validator-4.png"
+  alt="Downloading MyTonCtrl"
+/>
+
+### 2.2 Run validator installation
+
+Run the validator installation. Installation takes approximately 20 minutes:
+
+```shell
+#install validator for Mainnet
+sudo bash install.sh -m single-nominator -n mainnet
+```
+
+<Image
+  src="/resources/images/nodes/run-validator/run-validator-5.png"
+  darkSrc="/resources/images/nodes/run-validator/run-validator-5.png"
+  alt="Expected final output for validator installation"
+/>
+
+<Accordion
+  title="Install validator for Testnet"
+>
+  ```shell
+  #install validator for Testnet
+  sudo bash install.sh -m single-nominator -n testnet
+  ```
+</Accordion>
+
+### 2.3 Verify status results
+
+Launch MyTonCtrl with the command `mytonctrl`. Then, check synchronization using the `status` command:
+
+```sh
+mytonctrl
+MyTonCtrl> status
+```
+
+<Image
+  src="/resources/images/nodes/run-validator/run-validator-2.png"
+  darkSrc="/resources/images/nodes/run-validator/run-validator-2.png"
+  alt="MyTonCtrl status results"
+/>
+
+<Aside type="tip">
+  Initially, the status shows "Local validator initial sync status: downloading blockchain state x", which indicates that the node has started synchronization.
+  There are also other status messages in the very beginning of the synchronization, such as "starting", "init block seqno", "last key block is x", "downloading all shard states", "mc seqno" (Can be checked [here](https://github.com/ton-blockchain/ton/blob/master/validator/manager-init.cpp)) and all these statuses are fine.
```

Comment:
probably should not refer to C++ sources, actually we need to have this info in the doc (because the purpose of the doc is not to read c++ sources)